trento/xml/article_sap_trento.xml

```diff
@@ -2328,7 +2332,7 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
     <para>
       Any &suse; customer, with a registered &sles4sap;&nbsp;15 (SP1 or higher) distribution,
      experiencing an issue with &trentopremium;, can report the problem either directly in the &scc;
-     or through the corresponding vendor, depending on their licensing model. Any how, the problem has to reported under
+     or through the corresponding vendor, depending on their licensing model. Anyhow, the problem has to reported under
      &sles4sap;&nbsp;15 and component trento. 
```

Comment:
I think, here we want something from the user (to file a bug report), so I would address the user with "please" and make it sound not too harsh:

```suggestion
     or through the corresponding vendor, depending on their licensing model. Please report the problem under &sles4sap;&nbsp;15 and component trento.
```